src/components/RequirementView.vue

```diff
@@ -20,7 +22,7 @@
         <div class="separator"></div>
         <div
           v-for="(subReq, id) in req.ongoing"
-          :key="subReq.id">
+          :key="id">
```

Comment:
The old code is clearly wrong, probably due to some copy-pasta. subReq never contains an id. I also made some similar fixes elsewhere.